src/LoadableModules.h

```diff
@@ -9,11 +9,13 @@
 #ifndef SQUID_SRC_LOADABLEMODULES_H
 #define SQUID_SRC_LOADABLEMODULES_H
 
+#include "sbuf/forward.h"
+
 // TODO: add reporting for cachemgr
 // TODO: add reconfiguration support
 
-class wordlist;
-void LoadableModulesConfigure(const wordlist *names);
+/// list of library filenames to load dynamically
```

Comment:
The added description does not match the function it describes (it kind of matches its parameter instead).

```suggestion
/// dynamically load named libraries, in the listed order
```

or

```suggestion
/// Dynamically loads libraries identified by their filenames.
/// The libraries are loaded in the listed order.
```

or something like that.